src/Change.php

```diff
@@ -139,6 +139,29 @@ public function canCreateItem()
         return Session::haveRight(self::$rightname, CREATE);
     }
 
+    /**
+     * Overloaded from commonDBTM
+     *
+     * @since 0.83
```

Comment:
```suggestion
     * @since 10.0.19
```

Reply:
PHPDoc should not be added here at all as it blocks inheriting the PHPDoc from the base method and adds nothing.